src/features/user/userSlice.js

```diff
@@ -0,0 +1,51 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  id: "",
+  name: "",
+  email: "",
+  company: "",
+  location: {},
+  image: "",
+  price: "",
+  likes: [],
+  liked: [],
+  match: [],
+};
```

Comment:
```suggestion
     const user = {
        id: data.profile._id,
        name: data.profile.name,
        email: data.profile.email,
        company: data.profile.company,
        image: data.profile.image,
        likes: data.profile.likes,
        match: data.profile.match,
        requests: data.profile.incomingCoffeeRequest,
      };
```
App.js 에서 무엇을 받아오는지 확인해보시고 initial state 를 변경해주시면 좋을 것 같아요

Reply:
넵!